demo/client/src/3d-demo.ts

```diff
@@ -484,14 +489,27 @@ class ThreeDRenderer extends Renderer {
     }
 
     redrawVR(frame: VRFrameData): void {
+        this.clearDestFramebuffer(true);
         this.vrProjectionMatrix = frame.leftProjectionMatrix;
+        this.rightEye = false;
         this.camera.setView(F32ArrayToMat4(frame.leftViewMatrix), frame.pose);
         this.redraw();
+        this.rightEye = true;
         this.vrProjectionMatrix = frame.rightProjectionMatrix;
         this.camera.setView(F32ArrayToMat4(frame.rightViewMatrix), frame.pose);
         this.redraw();
     }
 
+    setDrawViewport() {
+        let offset = 0;
+        if (this.rightEye) {
+            offset = this.destAllocatedSize[0];
+        }
```

Comment:
nit: let's just write `const offset = this.vrRightEye ? this.destAllocatedSize[0] : 0;` in order to make `offset` immutable.